src/Maestro/tests/scenarios.md

```diff
@@ -0,0 +1,131 @@
+# Maestro++ Scenarios and test coverage
+
+## Overview
+
+This is a list of the business scenarios that Maestro is involved in, and the automated or manual validation to be performed for those scenarios
+
+## Package Publishing
+
+Maestro is only involved in the insertion of builds into the BAR.
+
+### Automated Tests
+Scenario tests for the latest version of the API are available at https://github.com/dotnet/arcade-services/blob/master/src/Maestro/tests/Scenarios/builds.ps1
+
+Unit tests for the 20190116 API version are available at: https://github.com/dotnet/arcade-services/blob/master/src/Maestro/Maestro.Web.Tests/BuildController20190116Tests.cs
+
+## Manual testing
```

Comment:
So what are the manual tests we should run?  Just the two post statement below?

Reply:
Yes. If creating a build works, then publishing should work.